app.rb

```diff
@@ -2,8 +2,9 @@
 require_relative 'teacher'
 require_relative 'student'
 require_relative 'rental'
+require_relative 'user_inputs'
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/201698543-72312559-900d-448d-a081-661065dc1227.png)

- [ ] You are writing **print** and **gets.chomp** a lot of times means you are repeating your code which is not making your code DRY. Kindly make your code DRY by making a function for taking inputs from the users :+1:

Hint: See the function below ⬇️
```
def inputs(show)
  outputs = []
  show.each do |s|
    print s
    outputs.push(gets.chomp)
  end
  outputs
end
```